spec/predicates/provenance.md

```diff
@@ -304,7 +304,8 @@ WARNING: This is only for demonstration purposes. The GitHub Actions team has
 not yet reviewed or approved this design, and it is not yet implemented. Details
 are subject to change!
 
-GitHub-Hosted runner:
+If GitHub is the one to generate provenance, and the runner is GitHub-hosted,
```

Comment:
I think this just brings up a detail that wasn't obvious to me: "Builder" primarily refers to the (id)entity generating the provenance, not the "runner" that actually did the build. This feels like it could easily be broken into two pieces to make things more clear (esp. at the lower SLSA levels): 1) authority that attests to the result of the build / content of provenance 2) runner that executes the build steps.

But given that that'd be a bigger change, I do think this is on the right path towards improving the examples.

Reply:
OK, I'll file an issue for that. It's a good point.